convert2rhel/actions/pre_ponr_changes/subscription.py

```diff
@@ -195,7 +195,15 @@ def run(self):
                     return
                 raise
 
-            logger.warning("No rhsm credentials given to subscribe the system. Skipping the subscription step.")
+            if subscription._is_registered() and not subscription.is_sca_enabled():
+                self.set_result(
+                    level="ERROR",
+                    id="SYSTEM_REGISTERED_WITHOUT_SCA",
+                    title="Registered with RHSM but without Simple Content Access (SCA) enabled",
+                    description="This system has been registered with Red Hat Subscription Manager but Simple Content Access is not enabled.",
+                    remediation="To resolve this error please enable Simple Content Access and run the conversion again.",
```

Comment:
I think it would be good to include some link to in the remediation(?) that show the user where they could neable the SCA. 

Also, don't forget to update the keyword here.
```suggestion
                    remediations="To resolve this error please enable Simple Content Access and run the conversion again.",
```

Reply:
Good point i'll add this